### DISCUSSION TITLE
[Photos Desktop] Storing audio files?

### DISCUSSION THREAD

**Discussion by taivlam** [ente-io/ente#3836] [Q&A] [answered]
Does Ente support audio-only/sound files?

I have quite a few audio recordings from meetings/interviews from my Zoom [H1n recorder](https://zoomcorp.com/en/gb/handheld-recorders/handheld-recorders/h1n-handy-recorder/) that are recorded in WAV format.

If so, I will consider  using Ente to store audio files as part of my backup procedures.

**Comment by crKRoEYS** [comment, marked as answer]
No they don't

"What file formats does Ente support?

Ente supports all files that have a mime type of image/* or video/* regardless of their specific format.

However, we only have limited support for RAW currently. We are working towards adding full support, and you can watch this [thread](https://github.com/ente-io/ente/discussions/625) for updates.

If you find an issue with ente's ability to parse a certain file type, please write to [support@ente.io](mailto:support@ente.io) with details of the unsupported file format and we will do our best to help you out."

https://help.ente.io/photos/faq/general
